keyboards/ergodox_ez/keymaps/default/keymap.c

```diff
@@ -80,23 +72,14 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
  */
 [SYMB] = LAYOUT_ergodox(
```

Comment:
```suggestion
[SYMB] = LAYOUT_ergodox_pretty(
```
No?

Reply:
Yup, I got the first layer but not the others.  Fixed now.